QGCPostLinkCommon.pri

```diff
@@ -105,11 +105,11 @@ LinuxBuild {
         libQt5TextToSpeech.so.5
 
     !contains(DEFINES, __rasp_pi2__) {
-        # Some Qt distributions link with *.so.56
+        # Some Qt distributions link with *.so
         QT_LIB_LIST += \
-            libicudata.so.56 \
-            libicui18n.so.56 \
-            libicuuc.so.56
```

Comment:
Just a comment, usually @DonLakeFlyer is not a huge fan of a non version specific dependencies, so that may be a problem.
Check the others PRs for here: https://github.com/mavlink/qgroundcontrol/pulls?q=is%3Apr+libicudata

Reply:
You reminded me. I just comment that lines out, and all build process is working fine. 
Seems my Qt distributions don't need link with that .so files, but I don’t know how to determine which distributions need them.

Reply:
> is not a huge fan of a non version specific dependencies, so that may be a problem.

This was a Qt thing where it required a specific version to work with. With latest 5.12 version that requirement may have changed. Don't know.